src/text/Tag.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+
+import "./Tag.scss"
+
+export interface TagProps {
+  /** Tag text */
+  children: React.ReactNode
+  /** Appearance of the component */
+  variant?:
+    | "primary"
```

Comment:
nit: might be nice to make this a type / enum!

Reply:
I think a type would make sense if we were sharing variant names across multiple components…otherwise, my preference is to have it inline with the rest of the props.